nature_importer.py

```diff
@@ -101,7 +143,7 @@ def load_mapping_files():
                     "items.json",
                     "properties.json"]
     for filename in files_to_get:
-        filename_base = os.path.splitext(os.path.basename(filename))[0]
+        filename_base = filename.split(".")[0]
```

Comment:
`os.path.splitext()` is safer to use since it will survive any "." earlier in the filename.